coverage/.last_run.json

```diff
@@ -1,5 +1,5 @@
 {
   "result": {
-    "covered_percent": 76.31
+    "covered_percent": 77.57
   }
 }
```

Comment:
The `/coverage` directory should not be included in your repository. You can remove it with:

```no-highlight
$ git rm -rf coverage
```

And then add it to your `.gitignore` file:

```no-highlight
/coverage
```